bodhi/server/services/graphql.py

```diff
@@ -0,0 +1,64 @@
+# Copyright © 2014-2019 Red Hat Inc., and others.
+#
+# This file is part of Bodhi.
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+"""Defines API endpoints related to GraphQL objects."""
+
+import math
+
+from cornice import Service
+from cornice.validators import colander_body_validator, colander_querystring_validator
+from graphene import ObjectType, Field, Schema, String
+from pyramid.exceptions import HTTPNotFound
+from sqlalchemy import func, distinct
+from sqlalchemy.sql import or_
+
+from bodhi.server import log, security
+from bodhi.server.models import (
+    Update,
+    UpdateStatus,
+    UpdateType,
+    UpdateRequest,
+    Build,
+    BuildrootOverride,
+    Package,
+    Release,
+    ReleaseState,
+    TestGatingStatus,
+)
+from bodhi.server.validators import (
+    validate_tags,
+    validate_enums,
+    validate_updates,
+    validate_packages,
+    validate_release,
+)
+import bodhi.server.schemas
+import bodhi.server.services.errors
+
```

Comment:
There are many imports that we don't really need now so we should remove them. You can run `flake8 bodhi/server/services/graphql.py` in vagrant and you will have a list of all the unused imports.